include/vcpkg/vcpkgpaths.h

```diff
@@ -175,7 +182,12 @@ namespace vcpkg
 
         // the directory of the builtin ports
         // this should be used only for helper commands, not core commands like `install`.
-        path builtin_ports_directory() const { return this->builtin_ports; }
+        path builtin_ports_directory() const { return this->root / u8path("ports"); }
+        path builtin_versions_directory() const { return this->root / u8path("versions"); }
```

Comment:
```suggestion
        path builtin_registry_ports_dir() const { return this->root / u8path("ports"); }
        path builtin_registry_versions_dir() const { return this->root / u8path("versions"); }
```

renaming these is good, imo.